Extending/Packages/Creating-a-Package/index.md

```diff
@@ -0,0 +1,146 @@
+| Version | 1.0.0 | This is automatically set to 1.0.0 but can be changed or increased manually if needed |
+| Icon Url | _Blank_ | Not a mandatory value, but will appear as the package icon different places in the backoffice if set |
+| Umbraco version | 8.2.0 | It will automatically select the version your Umbraco is currently on. This is then set as the compatible version for the package |
+| Author | Your name | Here you get to take credit for your awesome work! |
+| Author URL | Your website or maybe Twitter tag | Will link to this from the author name certain places in the backoffice |
+| Contributors | _Blank_ | Here you can add the names of other contributors if you have any |
+| License | MIT License | Will be set to MIT by default. If you are planning on selling your package you can change the license here |
+| License URL | http://opensource.org/licenses/MIT | Will be set to the URL for the MIT license, can be changed as needed |
+| Package readme | This will add a dashboard to your content section. <br> <br>The dashboard will show each user the most recent nodes they have saved. | This will be shown when someone looks at the package in the package dashboard |
+
+### The Package Content section
+
+This section is used to determine which things the package should contain. We will fill in the following things:
+
+| Property | Value | Note |
+| -------- | ----- | ---- |
+| Content | _Empty_ | Here you can include content - fx if you want to create a starter kit. Not relevant for this package though |
+| Document Types | _Empty_ | Similar to the Content picker above. Important to note that if you include content you will need to also pick all its dependencies in this and the next steps for it to be packaged together! |
+| Templates | _Empty_ | See `Document Types` above |
+| Stylesheets | _Empty_ | This will come from the ~/css folder. If you have stylesheets you want to include from other locations you can do so at a later step |
+| Macros | _Empty_ | See `Document Types` above |
+| Languages | _Empty_ | See `Document Types` above - all text is hardcoded or within our own lang folder in this package, so this is not needed |
+| Dictionary | _Empty_ | See `Document Types` above |
+| Data Types | _Empty_ | See `Document Types` above |
+
+### The Package Files section
+
+In this section you will be able to select all of your own custom files. Under the `Path to file` option. since everything in our Dashboard is from the same folder in App_Plugins, we can select the folder and it will include all items inside that folder:
+
+![Selecting the files you want in your package](images/select-files-for-package.png)
+
+We will leave the `Package options view` selector empty, but in case you were wondering you can select an html file here that will show up as package options. Here is an example from the package [Articulate](https://github.com/Shazwazza/Articulate/tree/master/src/Articulate.Web/App_Plugins/Articulate/BackOffice/PackageOptions):
```

Comment:
```suggestion
We will leave the `Package options view` selector empty, but in case you were wondering you can select an HTML file here that will show up as package options. Here is an example from the package [Articulate](https://github.com/Shazwazza/Articulate/tree/master/src/Articulate.Web/App_Plugins/Articulate/BackOffice/PackageOptions):
```